include/crm/common/output_internal.h

```diff
@@ -28,7 +28,7 @@ extern "C" {
  */
 
 
-#  define PCMK__API_VERSION "2.22"
+#  define PCMK__API_VERSION "2.24"
```

Comment:
This is so that when #2836 gets merged we won't forget to bump for this one too.